x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -19,54 +27,95 @@ import {
   getActivityLogError,
   getActivityLogIterableData,
   getActivityLogRequestLoaded,
+  getLastLoadedActivityLogData,
   getActivityLogRequestLoading,
 } from '../../store/selectors';
 
+const Sentinel = styled.div`
```

Comment:
Maybe 😄 
A more descriptive name? perhaps `LoadMoreTrigger`?

Reply:
Yeah. For some reason, I went with a more ominous-sounding name 😂

Reply:
done 7e962e4f300012c6d0b84d6b88bc241177478700